src/main/java/frc/team2412/robot/subsystem/ClimbSubsystem.java

```diff
@@ -146,11 +177,11 @@ public void angleClimbHook(DoubleSolenoid.Value value) {
     }
 
     public void extendFixedArm() {
-        setMotor(testSpeedExtend.getValue().getDouble(), climbFixedMotor);
+        setMotor(30 * ENCODER_TICKS_PER_INCH, climbFixedMotor);
     }
 
     public void retractFixedArm() {
-        setMotor(testSpeedRetract.getValue().getDouble(), climbFixedMotor);
+        setMotor(2 * ENCODER_TICKS_PER_INCH, climbFixedMotor);
```

Comment:
This and the `30` in `extendFixedArm()` should be constants and optionally configurable in Shuffleboard.